src/main/java/com/demodemo/demodemo/HelloController.java

```diff
@@ -3,11 +3,21 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 @RestController
 public class HelloController {
 
-@GetMapping("/hello")
-    public String hello(){
-    return  "Hello world!";
-}
+    @GetMapping("/fruit")
```

Comment:
```suggestion
    @GetMapping("/fruits")
```
REST APIのパスは名詞かつ複数形が望ましいです。